docs/flatpak-metadata/io.github.lavenderdotpet.LibreQuake.metainfo.xml

```diff
@@ -12,7 +12,7 @@
     <id>librequake.desktop</id>
   </provides>
   <name>LibreQuake</name>
-  <summary>Free content first-person shooter</summary>
```

Comment:
id suggest leaving this line as is 
or going with open source instead of free content
that or if u wanna stick to lovecraftian go with cosmic horror instead of using the name of lovecraft

Reply:
This is the line that fails **Summary: Understandable for a non-technical person**

https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#not-technical

I like the normal version as well just tried to get in line. Say the word an we'll leave it as is.

Reply:
heard if that's the case then let's go with cosmic horror

Reply:
I have changed it